src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+    if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The implementation uses hardcoded values for specific days, which is not scalable or maintainable. Instead, use a base daily rate and apply discounts based on the number of days rented. Define these values as constants at the beginning of the function.